frontend/src/pages/Roster.js

```diff
@@ -0,0 +1,173 @@
+
+  React.useEffect(() => {
+    const loadData = async () => {
+      const { ok, data } = await getUsers();
+      if (ok) {
+        const developers = data.users
+          .filter((user) => user.role.name !== "Alum")
+          .map((user) => ({
+            ...user,
+            role: user.role.name,
+          }));
+        const alums = data.users
+          .filter((user) => user.role.name === "Alum")
+          .map((user) => ({
+            ...user,
+            role: "Alum",
+          }));
+        setState((prev_state) => ({
+          ...prev_state,
+          loading: false,
+          developers,
+          alumni: alums,
+        }));
+      } else {
+        dispatch(openAlert(`Error: ${data.message}`));
+        setState((prev_state) => ({
+          ...prev_state,
+          loading: false,
+        }));
+      }
+    };
+    if (state.loading) {
+      loadData();
+    }
+  }, [state.loading, dispatch]);
+
+  return (
+    <>
+      <Helmet>
+        <title>TSE Roster</title>
+      </Helmet>
+      <PageContainer>
+        <LoadingContainer loading={state.loading}>
+          <Button
+            size="large"
+            style={alumni ? deafultButton : styleButton}
+            onClick={() => {
+              setAlumni(false);
+            }}
+          >
+            Active Members
+          </Button>
+          <Button
+            size="large"
+            style={!alumni ? deafultButton : styleButton}
+            onClick={() => {
+              setAlumni(true);
+            }}
+          >
+            Alumni
+          </Button>
+          <Grid container spacing={0} alignItems="center" justify="center" className={classes.grid}>
+            <Grid item xs={12}>
+              <MaterialTable
+                icons={{ ...TableIcons, Filter: TableIcons.Search }}
+                onRowClick={(evt, currRow) => setSelectedRow(currRow.tableData.id)}
+                options={{
+                  rowStyle: (rowData) => ({
+                    backgroundColor: selectedRow === rowData.tableData.id ? "#EEE" : "#FFF",
+                  }),
+                  filtering: true,
+                  paging: true,
+                  pageSize: 15,
+                  emptyRowsWhenPaging: true,
+                  pageSizeOptions: [15, 30, 60, 100],
+                  toolbar: false,
+                }}
+                columns={[
+                  {
+                    title: "Name",
+                    field: "name",
+                    cellStyle: {
+                      textAlign: "left",
+                      paddingLeft: "50px",
+                    },
+                    headerStyle: {
+                      textAlign: "left",
+                      paddingLeft: "50px",
+                    },
+                    filterCellStyle: {
+                      textAlign: "left",
+                      paddingLeft: "50px",
+                    },
+                  },
+                  {
+                    title: "Role",
```

Comment:
Can we make this display "Previous Role" when viewing alumni, like on the Figma?